go-controller/pkg/node/controllers/upgrade/node_upgrade.go

```diff
@@ -82,7 +82,7 @@ func (uc *upgradeController) GetTopologyVersion(ctx context.Context) (int, error
 	// Masters not yet upgraded, check Node objects
 	klog.Infof("Could not determine TopologyVersion via configmap, falling back to Nodes")
 
-	masterNode, err := labels.NewRequirement("node-role.kubernetes.io/master", selection.Exists, nil)
+	masterNode, err := labels.NewRequirement("node-role.kubernetes.io/control-plane", selection.Exists, nil)
```

Comment:
do u still need to look for both labels till we get to 1.25 ?

Reply:
The "control-plane" label will be there for sure in 1.24, so I believe we can get away with just checking that one. @trozet what's your take on this?